src/DataStructures/DenseMatrix.hpp

```diff
@@ -0,0 +1,45 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines class DenseMatrix.
+
+#pragma once
+
+#include <blaze/math/DynamicMatrix.h>
+#include <pup.h>  // IWYU pragma: keep
+
+#include "Utilities/Blaze.hpp"
```

Comment:
these need to be after the above line:
`#include <blaze/math/DynamicMatrix.h>`
`#include <blaze/system/Version.h>`